src/main/java/io/vertx/core/http/impl/HttpChannelConnector.java

```diff
@@ -191,7 +191,7 @@ private void applyHttp1xConnectionOptions(ChannelPipeline pipeline) {
       false,
       options.getDecoderInitialBufferSize()));
     if (options.isTryUseCompression()) {
-      pipeline.addLast("inflater", new HttpContentDecompressor(true));
+      pipeline.addLast("inflater", new HttpContentDecompressor());
```

Comment:
can we use `false` instead to be explicit and not rely on the default constructor ?

Reply:
I think you are right,This place should not depend on default。